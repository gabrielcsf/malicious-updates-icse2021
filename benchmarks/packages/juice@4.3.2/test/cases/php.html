<html><body>
<?php
echo "Hello world";
?>
<a href="<?php echo user()->getSystemAddress() ?>" target="_blank"></a>
</body></html>
