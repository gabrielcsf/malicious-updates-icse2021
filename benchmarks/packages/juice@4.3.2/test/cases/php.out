<html><body style="color: red;">
<?php
echo "Hello world";
?>
<a href="<?php echo user()->getSystemAddress() ?>" target="_blank"></a>
</body></html>
